docker_test.go

```diff
@@ -371,6 +450,7 @@ func run(m *testing.M) int {
 			"writeKey":    writeKey,
 			"workspaceId": workspaceID,
 			"postgresPort": resourcePostgres.GetPort("5432/tcp"),
+			"address": address,
```

Comment:
Following the suggestion above, 
```suggestion
			"redisHost": redisHost,
```